File: Python/004_Lista_if_else/Ex_013.py
"""
13) Faça um programa que leia a idade de um nadador e mostre em qual categoria ele é classificado, de
acordo com a tabela a seguir.
"""

idade = int(input("Digite a idade do usuario: "))

if 4 <= idade <= 5:
    print("Fraldinha")
elif 6 <= idade <= 12:
    print("Infantil")  
elif 13 <= idade <= 18:
    print("Juvenil")
else: 
    print("Adulto")

File: XDES01 - Linguagem C/004_Lista_if_else/Ex_013.c
//13) Faça um programa que leia a idade de um nadador e mostre em qual categoria ele é classificado, de acordo com a tabela a seguir.
#include <stdio.h>
int main ()
{
    int idade;

    printf("Digite sua idade: ");
    scanf("%d", &idade);

    if( idade > 3 && idade < 6 )
    {
        printf("Fraldinha");    
    }
        else
            if( idade > 5 && idade < 13 )
            {
                printf("Infantil");    
            }
                else
                    if( idade > 12 && idade < 19 )
                    {
                        printf("Juvenil");    
                    }
                        else
                            if( idade > 18 )
                            {
                                printf("Adulto");    
                            }
return 0;
}                                
